container/features/src/main/resources/features.xml

```diff
@@ -257,10 +257,15 @@
       <bundle>wrap:mvn:org.json/json/20140107$Export-Package=org.json</bundle>
     </feature>
 
-    <feature name="owasp-encoder" description="owasp-encoder" version="${owaspEncoderVersion}">
+    <feature name="owasp-encoder" description="OWASP :: Encoder" version="${owaspEncoderVersion}">
       <bundle>wrap:mvn:org.owasp.encoder/encoder/${owaspEncoderVersion}$Export-Package=org.owasp.encoder</bundle>
     </feature>
 
+    <feature name="owasp-html-sanitizer" description="OWASP :: HTML Sanitizer" version="${waspHtmlSanitizerVersion}">
```

Comment:
Shouldn't this be "${owaspHtmlSanitizerVersion}" ?

Reply:
Why, yes, yes it should. Nice catch.